deployment/clouddeploy/gke-workers/environments/oss-vdb-test/combine-to-osv.yaml

```diff
@@ -12,5 +12,7 @@ spec:
             env:
             - name: GOOGLE_CLOUD_PROJECT
               value: oss-vdb-test
+            - name: INPUT_GCS_BUCKET
```

Comment:
Probably want to set this in the production `oss-vdb/combine-to-osv.yaml` also

Reply:
It defaults when unset: https://github.com/google/osv.dev/blob/38a28cd7123e13f90206872cc2e9af57959cabfa/vulnfeeds/cmd/combine-to-osv/run_combine_to_osv_convert.sh#L11